pkg/cephmgr/osd/agent.go

```diff
@@ -70,10 +72,22 @@ func (a *osdAgent) Name() string {
 // set the desired state in etcd
 func (a *osdAgent) Initialize(context *clusterd.Context) error {
 
+	deviceCount := 0
 	if len(a.devices) > 0 {
-		// add the devices to desired state
-		devices := strings.Split(a.devices, ",")
+		var devices []string
+		if strings.EqualFold(a.devices, allDevices) {
```

Comment:
i hadn't come across EqualFold yet.  is that idiomatic golang for doing case insensitive comparisons?